projects/baby_driver/scripts/can_util.py

```diff
@@ -114,7 +114,6 @@ def send_message(
         data=data,
         is_extended_id=False
     )
-
```

Comment:
did you delete this accidentally?

Reply:
Yes I did, I was trying to see what the can message looked like, so I inserted a print statement there and deleted it, I may have deleted an extra line by accident